pipeline.yaml

```diff
@@ -9,6 +9,15 @@ resources:
       # This enables triggering on tag changes that match the glob pattern
       # tag_filter: "*.*.*"
 
+  - name: source-release
+    type: git
+    source:
+      uri: ((git_source))
+      private_key: ((privatekey))
+      branch: ((git_release_branch))
+      skip_ssl_verification: ((git_skip_ssl_verification))
+      tag_filter: "*.*.*"
+
```

Comment:
it's not used here. imho dead code.

Reply:
which one? `branch`?

Reply:
hmmm, the documentation i wrote in #24 , expects you to overwrite this file. we have to discuss that.

Reply:
as discussed offline the following actions will be done:
- remove pipeline.yaml and ci/ from bb-buildingblock
- modify example/ci/README.md to copy the example and modify for your needs

Reply:
Fixed. See 4e1eb78